src/io/flutter/view/FlutterView.java

```diff
@@ -234,6 +236,14 @@ private void addInspector(FlutterApp app, @Nullable InspectorService inspectorSe
       }
 
       addPerformanceTab(runnerTabs, app, !hasInspectorService);
+
+      // Track inspector tab selection analytics.
+      runnerTabs.addListener(new TabsListener.Adapter() {
```

Comment:
I'm not sure this is what we want because this fires gratuitously, for example, when the tab activates.  Really we just want explicit selections (which we get with a `TabsListener`).  (Actually, we may be able to infer that from previous selection --- if it's null, then that suggests the kind of selection we want.  I'll investigate and report back.)

Reply:
leaving the code as addListener makes sense in that case.  Maybe add a comment explaining why.
minor nit: use a lambda expression instead of an anonymous class.